holidays/countries/austria.py

```diff
@@ -19,6 +19,7 @@
 class Austria(HolidayBase, ChristianHolidays, InternationalHolidays):
     country = "AT"
     default_language = "de"
+    supported_categories = ("bank", "common")
```

Comment:
Not sure if `public` instead of `common` would be more appropriate here.

Reply:
I was thinking about it.